lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+# Creates a menu of takeaway items and displays menu
+class Menu
+  attr_reader :display , :items
+
+  def display
+    items.each do |dish,price|
+      "#{dish}: #{price}"
+    end
+  end
+  
+  def items
+    { "pasta" => 4.99, "bread" => 0.99, "pizza" => 5.99 } # => muttable
```

Comment:
This feels to me like it belongs in an instance variable rather than as a return value from a method.